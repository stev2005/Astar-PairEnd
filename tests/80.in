CATATGTCGTAAATTCTCAGGAATGGGCCTCGTCGGTCTCCAGCTCTGGTAATATCTCGAAAGTCCGAGCCTCTCACAGTTTATTTAGCTGCTC
15
AGTAAACAGCCGGCCCAACTTCAGGTATGCAGTTGAATAGACAGG
AGCCG
GGTACTATCACAATAAG
GTCGCTACGCTAGCGCGTCGGGGTGGGTGATA
CTCATGGACGTTAACTCTCTCAGAGCGTATGCGTGCGACTTAGTCTTGGAGACAATCGGCCCTTACAGCGGAGAATATG
TAGAAGACCGGGTCGAGGGAAA
TCAGCTATTAGTAATCCGAACCGCAAAACAAACCGGATGTTATAAGCG
CGGTATTATTCATCCAAACTGAGGTGGGAAGC
AAGATTTGATGGCGGCTCTATCTGTCGAATGAACATAA
ATAGAGACGCAGCCCTCTCCTATTAAAAAAGCTGTAAACGCGAGTGCACGCATC
TCTTGTCTTTGTAGAGAGTGGCAGGGAGCCCCGCAAAGT
CGGCATAACTTTATGGCGATTCACGAGTGCGATCCAACACATCAGTGACGTAATC
TT
GCAGACTACCCGCATGTGATTTGTTCCCAATGCTAGAAGCCTTGATATCCGCAAATCCACGTATGACGAATCTTATG
GTCAACACACGCTAATATTCCACCCCACAGGTCTTGCATCCCGAGTTGGTTTTCAA
