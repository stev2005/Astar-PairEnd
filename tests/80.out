21
1
4
13
36
10
22
13
18
27
15
25
0
37
26
